README.rst

```diff
@@ -104,6 +104,18 @@ The authentication micro-frontend also requires the following additional variabl
      - Disables the enterprise login from Authn MFE.
      - ``true`` | ``''`` (empty strings are falsy)
 
+   * - ``APP_ID``
+     - Enables support for runtime configurations that will be available in getConfig method specific to MFE
+     - ``true`` | ``''`` (empty strings are falsy)
```

Comment:
```suggestion
     - Name of MFE, this will be used by the API to get runtime configurations for the specific micro frontend.
     - ``frontend-app-authn`` | ``''`` (This name can be changed based on preference but the same name should be used at the backend for configurations. )
```

Reply:
@zainab-amir Runtime configuration follows the pattern `frontend-app-xyz` and use `xyz` as app id, that means app id for `frontend-app-authn` should be `authn`

Reply:
@syedsajjadkazmii thank you. Updating the suggestion:
```
     - Name of MFE, this will be used by the API to get runtime configurations for the specific micro frontend. For a frontend repo `frontend-app-appName`, use `appName` as APP_ID.
     - ``authn`` | ``''``
```

Reply:
addressed